presto-orc/src/main/java/com/facebook/presto/orc/TupleDomainFilterUtils.java

```diff
@@ -129,9 +131,18 @@ public static TupleDomainFilter toFilter(Domain domain)
                                 .toArray(byte[][]::new),
                         nullAllowed);
             }
+
+            if (isNotIn(ranges)) {
+                return BytesValuesExclusive.of(
+                        bytesRanges.stream()
+                                .map(BytesRange::getUpper)
```

Comment:
use existing `BytesRange::getLower` method here and remove newly added `getUpper`